src/memory/src/memory.cpp

```diff
@@ -67,7 +67,8 @@ uint8_t Memory::read(Address address) const {
 	// Sound Controller Registers
 	if (address_in_range(address, 0xFF26, 0xFF10)) {
 		// TODO: Sound Controller
-		Log::warn("Attempt to read from sound register " + num_to_hex(address));
+		//		Log::warn("Attempt to read from sound register " +
+		// num_to_hex(address));
```

Comment:
These commented out blocks can be removed

Reply:
Since the logs are now disabled, you can leave the warnings in place.

You can reset these changes in memory.cpp